### PR TITLE
Navigation Drawer Build Number UI - Shift to Nav Menu

### LINKED ISSUE #2794: [Landscape Mode] Navigation Drawer UI - Text Overlapping
Image

### PR DESCRIPTION
**Pull request by anandwana001**
Fixes #2794 

@shobhitagarwal1612 
Screenshot 2024-11-02 at 12 21 45 PM
Screenshot 2024-11-02 at 12 21 38 PM
 PTAL?

### CODE REVIEW

File: ground/src/main/res/layout/nav_drawer_footer.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright 2018 Google LLC
```
Comment (shobhitagarwal1612) [inline ground/src/main/res/layout/nav_drawer_footer.xml:4]: Please fix year

File: ground/src/main/res/menu/nav_drawer_menu.xml [hunk 2]
```diff
@@ -16,25 +16,35 @@
...
+      android:id="@+id/terms_of_service"
+      android:icon="@drawable/feed"
+      android:title="@string/terms_of_service" />
+  </group>
+  <group
+    android:id="@+id/nav_footer">
+    <item
+      android:id="@+id/nav_log_version"
+      android:title=""
+      app:actionLayout="@layout/nav_drawer_footer" />
```
Comment (shobhitagarwal1612) [inline ground/src/main/res/menu/nav_drawer_menu.xml:48]: We don't need this `actionLayout`. Can we remove it and cleanup the layout as well?

**Review by shobhitagarwal1612 — CHANGES_REQUESTED** [review]

File: ground/src/test/java/com/google/android/ground/ui/home/HomeScreenFragmentTest.kt [hunk 3]
```diff
@@ -162,7 +162,6 @@ class HomeScreenFragmentTest : AbstractHomeScreenFragmentTest() {
     swipeUpDrawer()
 
     onView(withId(R.id.terms_of_service)).check(matches(isEnabled()))
-    onView(withId(R.id.version_text)).check(matches(isDisplayed()))
```
Comment (shobhitagarwal1612) [inline ground/src/test/java/com/google/android/ground/ui/home/HomeScreenFragmentTest.kt:165]: Can we assert the build text in some other way?

Comment (anandwana001) [reply ground/src/test/java/com/google/android/ground/ui/home/HomeScreenFragmentTest.kt:165]: done

**Review by shobhitagarwal1612 — APPROVED** [review]

**Pull request merged**